Releasenotes.md

```diff
@@ -0,0 +1,77 @@
+# Releasenotes Haal-Centraal-BRP-historie-bevragen
+
+
+## Versie 1.0.0
+
+Alhoewel er nog geen eerdere officiële versie van de Haal-Centraal-BRP-hitorie-bevragen API als release is uitgebracht zijn de specificaties van deze enspoints als onderdeel van de specificatie van Haal-Centraal-BRP-bevragen het afgelopen jaar redelijk stabiel geweest.
+Voor het uitbrengen van release v1.0.0 is besloten om de historie-endpoints in een aparte API onder te brengen en daarvoor een eigen github-repository te maken. Daarbij zijn nieuwe inzichten die het afgelopen jaar zijn opgedaan opgenomen. Dit is een globaal (en niet limitatief) overzicht van wijzigingen die doorgevoerd zijn voordat v1.0.0 is uitgebracht. Enkele van deze wijzigingen zijn breaking ten opzichte van v0.9.0.
```

Comment:
kan je van "v0.9.0" link maken naar de specificaties in deze versie?

Reply:
Die v0.9.0 specificatie is de v0.9.0 van BRP-bevragen. Daarin zitten deze endpoints ook nog opgenomen. Ik zal dat specifiek melden in de releasenotes.